ASE2019-red/thingy-api-red
# Node Red workflow for Slack

To test:
1. Startup the whole docker stack with `docker-compose up -d`
2. Check if you can access http://localhost:1880 and see the configured workflow for Slack notifications
3. Check the following Slack channel for new messages: https://unifr-ase-19.slack.com/archives/CRLDQA51V (it's very noisy there, maybe you want to mute it)
  
Open stuff:
* Adaption of the websocket interval for coffee detection
* Variable usage in flows.json
